src/Misc/layoutbin/systemd.svc.sh.template

```diff
@@ -86,6 +84,15 @@ function install()
     chown ${run_as_uid}:${run_as_gid} ./runsvc.sh || failed "failed to set owner for runsvc.sh"
     chmod 755 ./runsvc.sh || failed "failed to set permission for runsvc.sh"
 
+    if [[ "${selinuxEnabled}" == "Enforcing" ]]; then
+        # We will need to add the initrc_exec_t context for the script
+        # (if we can't add, softfail first and try modify instead)
+        semanage fcontext --add    --type initrc_exec_t "${RUNNER_ROOT}/runsvc.sh" ||\
+        semanage fcontext --modify --type initrc_exec_t "${RUNNER_ROOT}/runsvc.sh" ||\
```

Comment:
_semanage_ doesn't appear to be installed on a stock Fedora system

Reply:
```
$ sudo chcon -t initrc_exec_t runsvc.sh
```
has worked locally. Of course that's only a temporary relabelling but wanted to let you know you've got the right context.

Reply:
Thanks @istathar , how annoying. CentOS and RedHat do ship with `semanage`. I guess the other option is to check for the context and bail if:
  1. Context is wrong; _and_
  2. no `semanage` is present

Reply:
I created a new CentOS 8 Azure VM and it doesn't come with `semanage` 😢